maintenance/generate-wikimedia-domain-config.js

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+
+/**
+ * Simple script to update sitematrix.json
+ */
+
+"use strict";
+var P = require('bluebird');
+
+var fs = require("fs"),
+	writeFile = P.promisify(fs.writeFile),
+	request = P.promisify(require('request')),
+	downloadUrl = "https://en.wikipedia.org/w/api.php?action=sitematrix&format=json",
+	filename = "sitematrix.json";
```

Comment:
Let's get rid of this chaining, pleaseee :P Apart from the fact that we do not use it, it looks rather ugly (to me).